### DISCUSSION TITLE
Hi, a contributor of LaTeX Workshop here

### DISCUSSION THREAD

**Discussion by tamuratak** [community/community#4738] [Codespaces] [answered]
You have raised some issues on LaTeX Workshop for GitHub Codespaces:

> The extension provides three kinds of features to help with LaTeX authoring: 1) a set of views that surface common commands, 2) a PDF previewer, and 3) language features like snippets and IntelliSense. The extension can be used fairly fully featured, but there are some web or security limitations.
> The following workarounds address limitations in the view and previewer feature areas:
>    The views themselves work without issue, but a few of the commands try to launch native applications, like revealing the output folder in the OS file explorer. These do nothing in the web, so using the desktop is an alternative.
    "latex-workshop.view.pdf.viewer" - This setting provides a PDF previewer similar to the Markdown preview. You can preview in a browser tab, in a separate PDF viewer, or in a VS Code tab.
>    Only the browser tab is usable in web-based Codespaces. The separate PDF viewer fails silently, and the VS Code tab hits the Content Security Policy issue. The VS Code Tab viewer has other features that aren't available due to this limitation.

https://code.visualstudio.com/docs/remote/codespaces#_some-extensions-behave-differently-in-the-web

We would like to investigate to solve the issues on our side. Could you give us access to GitHub Codespaces?

**Comment by tanmayeekamath** [comment, marked as answer]
@tamuratak Thanks for your feedback. We are aware that you would like to access GitHub Codespaces (and we're very excited to share it with you!) but we're being cautious with our rollout right now so I ask that you hang tight. We'll add more folks as soon as we can! Thanks for your patience.
